.i18n/Game-zh_CN.po

```diff
@@ -0,0 +1,5296 @@
+"symm at hb\n"
+"apply add_right_eq_self at hb\n"
+"apply add_right_eq_zero at hb\n"
+"rw [hb, add_zero]\n"
+"rfl\n"
+"```\n"
+"\n"
+"A passing mathematician remarks that with antisymmetry as well,\n"
+"you have proved that `≤` is a *partial order* on `ℕ`.\n"
+"\n"
+"The boss level of this world is to prove\n"
+"that `≤` is a total order. Let's learn two more easy tactics\n"
+"first.\n"
+msgstr ""
+"\n"
+"这是我的证明：\n"
+"```\n"
+"cases hxy with a ha\n"
+"cases hyx with b hb\n"
+"rw [ha]\n"
+"rw [ha, add_assoc] at hb\n"
+"symm at hb\n"
+"apply add_right_eq_self at hb\n"
+"apply add_right_eq_zero at hb\n"
+"rw [hb, add_zero]\n"
+"rfl\n"
+"```\n"
+"\n"
+"\n"
+"一位路过的数学家评论说，你已经运用“反对称性”证明了 `≤` 在 `ℕ` 上是一个*偏序*。\n"
+"\n"
+"这个世界中的 Boss 关卡是证明 `≤` 是一个全序。我们先学习两个更简单的策略。\n"
+
+#: Game.Levels.LessOrEqual.L07or_symm
+msgid "Dealing with `or`"
+msgstr "处理 `or`"
+
+#: Game.Levels.LessOrEqual.L07or_symm
+msgid ""
+"# Summary\n"
+"The `left` tactic changes a goal of `P ∨ Q` into a goal of `P`.\n"
+"Use it when your hypotheses guarantee that the reason that `P ∨ Q`\n"
+"is true is because in fact `P` is true.\n"
+"\n"
+"Internally this tactic is just `apply`ing a theorem\n"
+"saying that $P \\implies P \\lor Q.$\n"
+"\n"
+"Note that this tactic can turn a solvable goal into an unsolvable\n"
+"one."
+msgstr ""
+"# 小结\n"
+"`left` 策略将目标 `P ∨ Q` 转换为目标 `P`。当您的假设保证 `P ∨ Q` 为真的原因是由于 `P` 为真时，请使用它。\n"
+"\n"
+"在内部，这个策略只是应用了一个定理，该定理是 $P\\implies P\\lor Q$ 。\n"
+"\n"
+"请注意，此策略可能会将一个可解决的目标转换为不可解决的目标。"
+
+#: Game.Levels.LessOrEqual.L07or_symm
+msgid ""
+"# Summary\n"
+"The `right` tactic changes a goal of `P ∨ Q` into a goal of `Q`.\n"
+"Use it when your hypotheses guarantee that the reason that `P ∨ Q`\n"
+"is true is because in fact `Q` is true.\n"
+"\n"
+"Internally this tactic is just `apply`ing a theorem\n"
+"saying that $Q \\implies P \\lor Q.$\n"
+"\n"
+"Note that this tactic can turn a solvable goal into an unsolvable\n"
+"one."
+msgstr ""
+"＃ 小结\n"
+"`right` 策略将 `P ∨ Q` 的目标更改为 `Q` 的目标。\n"
+"当您的假设`Q` 为真是 `P ∨ Q`为真的原因时使用它。\n"
+"\n"
+"在策略内部，它只是 `apply` （应用） 了 $Q \\implies P \\lor Q$ 这个定理\n"
+"\n"
+"请注意，这种策略可以将可解决的目标变成无法解决的目标。"
+
+#: Game.Levels.LessOrEqual.L07or_symm
+msgid ""
+"\n"
+"Totality of `≤` is the boss level of this world, and it's coming up next. It says that\n"
+"if `a` and `b` are naturals then either `a ≤ b` or `b ≤ a`.\n"
+"But we haven't talked about `or` at all. Here's a run-through.\n"
+"\n"
+"1) The notation for \"or\" is `∨`. You won't need to type it, but you can\n"
+"type it with `\\or`.\n"
+"\n"
+"2) If you have an \"or\" statement in the *goal*, then two tactics made\n"
+"progress: `left` and `right`. But don't choose a direction unless your\n"
+"hypotheses guarantee that it's the correct one.\n"
+"\n"
+"3) If you have an \"or\" statement as a *hypothesis* `h`, then\n"
+"`cases h with h1 h2` will create two goals, one where you went left,\n"
+"and the other where you went right.\n"
+msgstr ""
+"\n"
+"\"`≤`的完全性是这个世界的老大级别，接下来就是它了。它表明如果`a`和`b`是自然数，\n"
```

Comment:
Should translate "boss level" consistently.

Reply:
I will fix it.